docs/assets/scss/common/_shortcodes.scss

```diff
@@ -0,0 +1,83 @@
+$padding-16: 1rem !default;
```

Comment:
Since this is file is all custom CSS that has to be manually injected into the theme, maybe it would be cleaner if we add the small edits from _code.scss and _app.scss here and then have it load at the end. That way all of our custom css that we're adding to theme is in one place (and maybe rename the file _custom.scss).

Reply:
So I merged these into a single file, but we still have to override _app.scss because that's what brings them altogether.